.github/workflows/sim.yml

```diff
@@ -14,11 +14,23 @@ jobs:
       - uses: actions/setup-go@v2.1.4
         with:
           go-version: 1.18
-      - name: Display go version
-        run: go version
+      -
```

Comment:
Is the newline intended here?

Reply:
Actually we should be changing all the ymls/adding linters.

Nicco made a standard at one point to have a new line after every `-` line, for easier copy/paste

But we didn't enforce it, so it fell off